app/views/second_level_browse_page/_new_second_level_browse_pages.html.erb

```diff
@@ -22,17 +22,7 @@
 
   <%# Generate accordion shaped markup %>
   <% if curated_order %>
-    <%
-      additional_contents = {
-        heading: {
-          text: list["title"],
-        },
-        content: {
-          html: contents,
-        },
-      }
-      accordion_contents << additional_contents
-      %>
+    <% accordion_contents = accordion_items(list["title"], accordion_contents, contents) %>
```

Comment:
If assignment is taking place here still, maybe the concatenation could just happen here instead of it being in the helper function.

Reply:
I've simplified the code a bit to remove the need to do the assignment in the template, have a look.

Reply:
yeah that's looking better!

Reply:
yeah that's looking better!